src/implementations/twig/components/link/link.story.js

```diff
@@ -26,86 +25,125 @@ const withParagraph = (story) => {
   return typeof demo === 'string' ? storyAsString(demo) : storyAsNode(demo);
 };
 
-const iconsList = {};
-iconsList.none = '';
-
-uiIcons.forEach((icon) => {
-  iconsList[icon] = icon;
-});
+const iconsList = uiIcons.reduce((a, b) => {
+  a[b] = b;
+  return a;
+}, {});
+
+const getArgTypes = (data) => {
+  return {
+    label: {
+      name: 'label',
+      type: { name: 'string', required: true },
+      defaultValue: data.link.label,
+      description: 'The main label of the button',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Link',
```

Comment:
This category is called "Content" in other PR. 
I think it's better to keep the same terminology everywhere

Reply:
Not sure it makes sense as "content", i believe these labels should reflect somehow the component, or "object" they belong to.